src/components/PizzaCanvas/PizzaCanvas.js

```diff
@@ -0,0 +1,126 @@
+import Component from '../../Component';
+import {PizzaImagery} from "../../services/PizzaImagery";
+import {CANVAS} from './settings';
+
+/**
+ * Class representing pizza constructed image.
+ * host is a <canvas> element
+ */
+export default class PizzaCanvas extends Component {
+  constructor(props) {
+    super(props);
+    this.ctx = this.host.getContext("2d");
+
+    this.host.width = CANVAS.px.width;
+    this.host.height = CANVAS.px.height;
+  }
+
+  /**
+   * Render view
+   * @returns {HTMLElement|string}
+   */
+  render() {
```

Comment:
In this case setting draw function as component render method is ok, cause we don't have animations on canvas. But what if we add, for instance, rotation animation, would it work properly? Just for future. Don't change anything here. :)